examples/iot/index.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+import * as iot from "@pulumi/azure/iot";
+
+const location = "West US 2";
+
+const resourceGroup = new azure.core.ResourceGroup("test", {
+    location: location,
+});
+
+const iotHub = new iot.IoTHub("test", {
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        capacity: 1,
+        name: "S1",
+        tier: "Standard",
+    },
+    fallbackRoute: {
+        source: "DeviceMessages", 
+        enabled: true, 
+        endpointNames: ["events"],
+        condition: "true"
+    }
+});
+
+iotHub.onEvent("test", async (context, arg) => {
```

Comment:
Nit: rename `arg` to `msg` or `message` for better readability

Reply:
fixed in 5a91398c8dfdbba830a2d789bf0c806b5933b32f